### PR TITLE
Show partition index for a offset on the topic page

### PR DESCRIPTION
**Pull request by PolatEmre-hg**
This Pull Request implements the feature of showing on the single topic page, which offset belongs to which partition. You can see this information by clicking on the "show topic button".

### CODE REVIEW

File: ui/src/app/layout/topics/single-topic.component.ts [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+} from '../../shared/services/topics.service';
+import {combineLatest, Observable} from 'rxjs';
+import {finalize, flatMap, map, shareReplay, startWith, take} from 'rxjs/operators';
+import {ApplicationsService, UserApplicationInfo} from '../../shared/services/applications.service';
+import {EnvironmentsService, KafkaEnvironment} from '../../shared/services/environments.service';
+import {ToastService} from '../../shared/modules/toast/toast.service';
+import {TranslateService} from '@ngx-translate/core';
+import {NgbDateStruct, NgbModal} from '@ng-bootstrap/ng-bootstrap';
+import {CertificateService} from '../../shared/services/certificates.service';
 import * as moment from 'moment';
```
Comment (albrechtflo-hg) [inline ui/src/app/layout/topics/single-topic.component.ts:19]: Please adjust your formatter to not reformat these imports.

File: ui/src/app/shared/services/topics.service.ts [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+    ApplicationInfo,
+    ApplicationsService,
+    BusinessCapabilityInfo,
+    UserApplicationInfo
+} from './applications.service';
+import {HttpClient} from '@angular/common/http';
+import {concatMap, map, take} from 'rxjs/operators';
+import {jsonHeader, ReplayContainer} from './services-common';
+import {combineLatest, forkJoin, Observable, of} from 'rxjs';
+import {EnvironmentsService, KafkaEnvironment} from './environments.service';
```
Comment (albrechtflo-hg) [inline ui/src/app/shared/services/topics.service.ts:12]: See above

**Review by albrechtflo-hg — CHANGES_REQUESTED** [review]

File: ui/src/app/layout/topics/single-topic.component.html [hunk 3]
```diff
@@ -370,7 +370,7 @@ <h4 class="modal-title">{{ 'View Topic Data' | translate }}</h4>
             <table class="table table-striped">
                 <thead>
                 <tr>
-                    <th>{{ 'Offset' | translate }}</th>
+                    <th>{{ 'Partition/Offset' | translate }}</th>
```
[the remark this answers is not in the document]
Comment (PolatEmre-hg) [reply ui/src/app/layout/topics/single-topic.component.html:373]: is adjusted now.

File: ui/src/app/layout/topics/single-topic.component.html [hunk 4]
```diff
@@ -385,7 +385,7 @@ <h4 class="modal-title">{{ 'View Topic Data' | translate }}</h4>
                 </tr>
                 <ng-container *ngIf="!topicDataLoading && (topicData | async)?.length">
                     <tr *ngFor="let row of topicData | async">
-                        <td>{{ row.offset }}</td>
+                        <td>{{ row.partition + '/' + row.offset }}</td>
```
[the remark this answers is not in the document]
Comment (PolatEmre-hg) [reply ui/src/app/layout/topics/single-topic.component.html:388]: Also adjusted .

File: ui/src/app/layout/topics/single-topic.component.ts (hunk 1 above)
Comment (PolatEmre-hg) [reply ui/src/app/layout/topics/single-topic.component.ts:19]: done.

File: ui/src/app/shared/services/topics.service.ts (hunk 2 above)
Comment (PolatEmre-hg) [reply ui/src/app/shared/services/topics.service.ts:12]: also done.

**Review by albrechtflo-hg — APPROVED** [review]

**Pull request merged**